certbot-ci/certbot_integration_tests/certbot_tests/assertions.py

```diff
@@ -1,6 +1,6 @@
 """This module contains advanced assertions for the certbot integration tests."""
 import io
-import os
+import os  # pylint: disable=os-module-forbidden
```

Comment:
This disable should not be needed anymore.

Reply:
`certbot-ci/certbot_integration_tests/certbot_tests/assertions.py:3:0: E5001: Forbidden use of os module, certbot.compat.os must be used instead (os-module-forbidden)`